### DISCUSSION TITLE
Magic link login

### DISCUSSION THREAD

**Discussion by DominusKelvin** [sailscastshq/boring-stack#45] [Ideas]
What do you all think of supporting magic link login for the authentication offering of The Boring JavaScript Stack?

**Comment by franklin-tina** [comment]
I'm personally not a big fan of magic links.
However, different projects have different requirements. 
If it will increase DX and make it easy for devs to integrate, I'll say go for it.

**Comment by Emmo00** [comment]
is it possible to implement the Scaffolding in a way that the users can choose whether they want it or not?

**Reply by DominusKelvin** [reply]
I'm not sure 🤔 

It's just code and you can always delete what you don't want
